.travis.yml

```diff
@@ -5,6 +5,7 @@ rvm:
 
 before_install:
 - git clone https://github.com/puppetlabs/puppetlabs-ruby_task_helper ../ruby_task_helper
+- git clone https://github.com/puppetlabs/puppetlabs-ruby_plugin_helper ../ruby_plugin_helper
```

Comment:
Looks like there is a rubocop problem, the spec tests are passinjg locally when i do some voodoo to get the plugin helper in the expected location. I think that if it is not too much trouble we could add another pre-install step with something like
```
- cd ruby_plugin_helper && git fetch origin pull/1/head:GH-1325 && git checkout GH-1325
```
just to verify everything is working as expected in travis.

Reply:
I was just going to wait until the plugin helper was merged and rerun travis. That's why this is marked 'Do not merge'